apps/block_scout_web/lib/block_scout_web/controllers/api/rpc/stats_controller.ex

```diff
@@ -71,6 +71,12 @@ defmodule BlockScoutWeb.API.RPC.StatsController do
     render(conn, "totaltransactions.json", count: transaction_estimated_count)
   end
 
+  def pendigcelo(conn, _params) do
```

Comment:
should be `pendingcelo` or `pending_celo` (missing the second letter "n" in pending)

Reply:
oh oops! thanks for catching that

Reply:
I used flatcase because almost all of the other methods in there are in flatcase